cmake/modules/FindGMock.cmake

```diff
@@ -77,6 +77,7 @@ if(NOT GMock_FOUND)
           EXCLUDE_FROM_ALL)
       endif()
       set(GMOCK_LIBRARIES gmock_main)
+      set(GMOCK_INCLUDE_DIRS ${GMOCK_SRC_DIR}/include)
```

Comment:
+1
I'm working on migrating our fork to Noetic, this was needed there too.

Reply:
Did you try installing the `libgmock-dev` package? I added Ubuntu Focal (which Noetic is primarily targeted at) to CI recently and it passes.

Reply:
So installing `libgmock-dev` does make it work.  But I actually still think this is a bug, and installing `libgmock-dev` just works around it.

When you install `libgmock-dev`, it means you actually don't come in the `if(NOT GMOCK_LIBRARIES)` block at all, and hence don't build gmock from source.  If for some reason you *do* need to build gmock from source, then this block still doesn't export the GMOCK_INCLUDE_DIRS properly.

Thus, I think we should either keep this fix, or we should remove the fallback to build from source completely.